101-recovery-services-backup-file-share/azuredeploy.json

```diff
@@ -132,49 +125,50 @@
     "backupManagementType": "AzureStorage",
     "containerType": "StorageContainer",
     "protectedItemType": "AzureFileShareProtectedItem",
+    "scheduleRunTimes": [ "[concat('2020-01-01T', parameters('scheduleRunTime'), ':00Z')]" ],
```

Comment:
sorry - still confused on this... is this a one time start time or a periodic time - the date suggests the former the description suggests the later...

Reply:
It is a periodic time. The scheduleRunTimes value must be of the [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) UTC format, which consists of both the date as well as the time. During policy creation, the date is ignored, and only the HH:MM value is used. The hardcoded date is therefore just a dummy prefix to make it compliant with the required format.